bazel-mode.el

```diff
@@ -21,6 +21,31 @@
 ;;
 ;;; Code:
 
+(defcustom buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'string
+  :group 'bazel-mode)
```

Comment:
This group needs to be defined first. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Group-Definitions.html.

Reply:
Done. That was dumb.